docker/Makefile

```diff
@@ -110,3 +110,11 @@ clean: remove-all
 	docker image rm athenz-conf || true
 	docker image rm athenz-setup-env || true
 	docker image rm rdl-athenz-java-model || true
+
+remove-local: remove-all
+	docker image rm abvaidya/athenz-setup-env:1.9.22 || true
```

Comment:
Noticed your username (abvaidya) is part of the path. Is that intentional?

Reply:
Yes. For now the docker images are uploaded to my personal docker account. Once we test it enough and agree , I can upload the same to Athenz docker account and change references to Athenz docker.

Reply:
why not use something like $USER instead?